my-burger-app/src/components/controls.jsx

```diff
@@ -0,0 +1,74 @@
+import { useState } from 'react';
+
+import minus from '../assets/minus_circle.svg';
+import plus from '../assets/plus_circle.svg';
+
+const mockData = [
```

Comment:
You are duplicating the same data as in `content.jsx`. Why? Use only 1 source of data and where you need it - just pass as a props